models/orderDao.js

```diff
@@ -0,0 +1,168 @@
+const {appDataSource} = require('./datasource')
+const middleErr = require('../middleware/error')
+
+// 장바구니에 담긴 상품 보여주기
+const busket = async (verifiedEmail) => {
+    try {
+        const userId = await appDataSource.query(`
+        select id from users where email = '${verifiedEmail}'
+        `)
+        const extractedId = userId[0].id
+        
+        const result = await appDataSource.query(`
+        SELECT p.id AS product_id, p.price AS product_price, p.name AS product_name, b.logo AS product_logo, bsk.quantity AS basket_quantity, b.name AS brand_name
+        FROM users u
+        JOIN basket bsk ON u.id = bsk.user_id
+        JOIN products p ON bsk.product_id = p.id
+        JOIN brands b ON p.brand_id = b.id
+        JOIN pay_status ps ON bsk.status_id = ps.id
+        WHERE u.id = '${extractedId}' AND ps.id = 1;       
+        `)
+        return result
+        
+    } catch (err) {
+        console.log(err)
+        middleErr.error(500 , 'INVALID_DATA_INPUT' )
+    }
+}
+
+//장바구니 상품 수량추가
+const addProduct = async (verifiedEmail, productId) => {
+    try {
+        const userId = await appDataSource.query(`
+        select id from users where email = '${verifiedEmail}'
+        `)
+        const extId = userId[0].id
+        const product_id = await appDataSource.query(`
+        select id from products where id = '${productId}'
+        `)
+        const extProduct_id = product_id[0].id
+        
+         const result =  await appDataSource.query(`
+                UPDATE basket
+                SET basket.quantity = basket.quantity + 1
+                WHERE basket.user_id = '${extId}' and basket.product_id = '${extProduct_id}'
+        `)
+    } catch (err) {
+        console.log(err)
+        const error = new Error('INVALID_DATA_INPUT')
+        error.statusCode = 500;
+        throw error
+    }
+};
+// 장바구니 상품 삭감
+const cutProduct = async(verifiedEmail,productId) => {
+    try{
+        const userId = await appDataSource.query(`
+        select id from users where email = '${verifiedEmail}'
+        `)
+        const extractedId = userId[0].id
+        const product_id = await appDataSource.query(`
+        select id from products where id = '${productId}'
+        `)
+        const extProduct_id = product_id[0].id
+        await appDataSource.query(`
+        update basket
+        set basket.quantity = basket.quantity - 1
+        where basket.user_id = '${extractedId}' and basket.product_id = '${extProduct_id}'
+        `)
+    
+    }catch(err){
+        console.log(err)
+        middleErr.error(500 , 'INVALID_DATA_INPUT' )
+    }
+};
+
+//장바구니 상품 삭제
+const deleteProduct = async(verifiedId, productId)=>{
```

Comment:
```suggestion
const deleteProduct = async(userId, productId)=>{
```
- verifiedId가 무슨 의미인지 명확하지 않습니다